
<html>
<head>
<script>
var testFunc = function() {
            }));
            var o = t("q1tI"),
                _ = t.n(o),
                u = t("uIZJ"),
                a = t("J0yO");

            function withRootErrorBoundary(r) {
                return Object(a.a)(r, _.a.createElement(u.a, null))
            }
</script>
</head>
<body>
<div id="res">Failed</div>
<script>
let failed = true;
try {
    testFunc();
    failed = false;
} catch (e) {
    failed = true;
}

document.getElementById("res").innerHTML = failed ? "Failed" : "Ok";
</script>
</body>
</html>
